Samsung/LPVS
# Add result page.

## Description

LPVS Front-End `Reseult` Page.
Result Page provides `PR informatin` and `detected file list`.

Dashboard(last page) pages will be uploaded.

## Type of change


# How Has This Been Tested?
1. Update environment variables of `docker-compose.yml` file.
2. Build the lpvs container : `docker compose build`
3. Run containers: `docker compose up`
4. See the lpvs home page with any browser: `http://localhost:7896`

For using social login with Naver, Kakao, and Google, fill the necessary information:
- Update the `application.properties` located in `src/main/resources`:
Enter client-id and client-secret. (Kakao doesn't require a client-secret.)
- Update the `.env` file located in `frontend`:
Enter your REST_API_KEY.

The values required to implement the login function can be obtained from the link below.
Naver: https://developers.naver.com/products/login/api/api.md
Kakao: https://developers.kakao.com/docs/latest/ko/kakaologin/common
Google: https://cloud.google.com/identity-platform/docs/web/google

![result](https://github.com/Samsung/LPVS/assets/57388014/86486d99-51f1-4926-95f3-f7273e8dfdad)
![1](https://github.com/Samsung/LPVS/assets/57388014/338cae5c-73f0-435a-8473-75ca2fae65f0)
![image](https://github.com/Samsung/LPVS/assets/57388014/1c414210-1620-4a3a-ba00-5219fb3283b0)


**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:
